doc/asciidoc/producer/configuration.adoc

```diff
@@ -32,4 +36,79 @@ all the messages sent to topics that don't exist are discarded;
 this because the `KafkaProducer.send()` method blocks the execution, as explained in https://issues.apache.org/jira/browse/KAFKA-3539[KAFKA-3539].
 You can tune the custom property `kafka.topic.discovery.polling.interval` in order to
 periodically check for new topics into the Kafka cluster so the plugin will be able
-to send events to the defined topics.
\ No newline at end of file
+to send events to the defined topics.
+
+==== Multi Database Support
```

Comment:
Great section here.  Please add answer to one question:

Will sink to / produce from fail if the database doesn't already exist?

Say I produce from foobar.  But I start a cold instance with nothing, and I want to create foobar later.  Does this work or does the whole thing die on startup?

This is not a requirement for the first version of 4.0 support but I bet people will ask for this, because we force peopel to pre-configure with neo4j.conf.  So imagine setting up a new machine.  It's a pain to start the cluster, create databases, shut down, reconfigure, restart, and then have your kafka.  It would be nice to configure just as you want, then start the cluster, then create the databases necessary.

Whichever way is the answer, just make it clear here in the docs

Reply:
Great question, we can change the `StreamsConfig` to `DATABASE` extension so it will be loaded at runtime for each database instance, this should allow us to permit to the user to "update" the `neo4j.conf` file before creating a database and then load the new configuration for the instance. Wdyt?

Reply:
that sounds workable.  :)